restli-tools/src/main/java/com/linkedin/restli/tools/clientgen/fluentspec/SpecUtils.java

```diff
@@ -120,33 +118,24 @@ public static String nameCamelCase(String name)
 
   public static String restMethodToClassPrefix(String name)
   {
-    ResourceMethod method = ResourceMethod.fromString(name);
-    switch (method)
+    switch (name)
```

Comment:
I thought you can just use name.toUpperCase()

Reply:
It doesn't matter as we are comparing the string directly.